Source/Extensions/Blazorise.Snackbar/Enums/SnackbarLocation.cs

```diff
@@ -0,0 +1,59 @@
+﻿#region Using directives
+using System;
+#endregion
+
+namespace Blazorise.Snackbar;
+
+/// <summary>
+/// Defines the snackbar location.
+/// </summary>
+public enum SnackbarLocation
+{
+    /// <summary>
+    /// Default behavior.
+    /// </summary>
+    [Obsolete( "Use Bottom instead." )]
+    Default,
+
+    /// <summary>
+    /// Show the snackbar on the bottom side of the screen.
+    /// </summary>
+    Bottom,
+
+    /// <summary>
+    /// Show the snackbar on the bottom-left side of the screen.
+    /// </summary>
+    BottomStart,
+
+    /// <summary>
+    /// Show the snackbar on the bottom-right side of the screen.
+    /// </summary>
+    BottomEnd,
+
+    /// <summary>
+    /// Show the snackbar on the left side of the screen.
+    /// </summary>
+    [Obsolete( "Use BottomStart instead." )]
+    Start,
+
+    /// <summary>
+    /// Show the snackbar on the right side of the screen.
+    /// </summary>
+    [Obsolete( "Use BottomEnd instead." )]
+    End,
+
+    /// <summary>
+    /// Show the snackbar on the top side of the screen.
+    /// </summary>
+    Top,
+
+    /// <summary>
+    /// Show the snackbar on the top-left side of the screen.
```

Comment:
the documentation refers to left & right. it should refer to start or end. we should not assume the left or right position when naming Start/End.

Reply:
So you think it should be Top-Start?

Reply:
I personally do. In the future we plan to have RTL support. So in that case the doc would be incorrect.